src/lib/nlcDb.js

```diff
@@ -27,43 +27,44 @@ const pouch = new Promise((resolve, reject) => {
 		this.db = db;
 
 		const syncFn = function(){
-			logger.info(`${TAG}: Starting sync of NLC training data with Cloudant.`);
-			// TODO: Commenting because we don't currently use this, but auto-training should be a future enhancement.
-			// let update = false;
-			db.sync(`https://${env.cloudantKey}:${env.cloudantPassword}@${env.cloudantEndpoint}/${env.cloudantDb}`,
-				{
-					include_docs: true,
-					filter: function(doc) {
-						// filter client side documents that we don't want synchronized
-						return doc.storageType !== 'private';
-					}
-				})
-				// .on('change', function(change){
-				// 	update = true;
-				// })
-				.on('complete', function(info){
-					logger.info(`${TAG}: Completed sync of NLC training data with Cloudant.`);
-					logger.debug(`${TAG}: Cloudant sync results.`, info);
+			if (!env.text && env.cloudantEndpoint && env.cloudantPassword) {
```

Comment:
should this be `env.test`  --> https://github.com/ibm-cloud-solutions/hubot-ibmcloud-cognitive-lib/blob/master/src/lib/env.js#L28